openfisca_uk/variables/demographic/person/age.py

```diff
@@ -0,0 +1,39 @@
+from openfisca_uk.model_api import *
+
+
+class age(Variable):
+    value_type = int
+    entity = Person
+    label = "Age"
+    unit = "year"
+    documentation = "The age of the person in years"
+    definition_period = YEAR
+
+    def formula(person, period):
+        ADULT_DEFAULT_AGE = 18
```

Comment:
should these be parameters?

Reply:
I'm leaning on no but not by that much - they're not policy...